events/wc_events/wc_central_kalimdor_events.txt

```diff
@@ -0,0 +1,1614 @@
+						}
+					}
+				}
+				modifier = {
+					add = {
+						if = {
+							limit = { 
+								intrigue >= 20
+							}
+							add = 85
+						}
+					}
+				}
+				custom_tooltip = tauren.53.b_winSkillCheck
+				trigger_event = tauren.561
+			}
+			35 = { # failure
+				compare_modifier = {
+					value = scope:duel_value
+					multiplier = -3.5
+					min = -49
+				}
+				custom_tooltip = tauren.53.b_loseSkillCheck
+				trigger_event = tauren.562
+			}
+		}
+	}
+	
+	#Threaten the bandits
+	option = {
+		name = tauren.53.c
+		
+		#Diplomacy Skill Check
+		duel = {
+			skills = { diplomacy }
+			target = scope:shoeless_bandit
+			50 = { # success
+				compare_modifier = {
+					value = scope:duel_value
+					multiplier = 3.5
+					min = -49
+				}
+				modifier = {
+					add = {
+						if = {
+							limit = { 
+								has_lifestyle = diplomacy_lifestyle
+							}
+							add = 10
+						}
+					}
+				}
+				modifier = {
+					add = {
+						if = {
+							limit = { 
+								diplomacy > 10
+								diplomacy < 20
+							}
+							add = 25
+						}
+					}
+				}
+				modifier = {
+					add = {
+						if = {
+							limit = { 
+								diplomacy >= 20
+							}
+							add = 85
+						}
+					}
+				}
+				custom_tooltip = tauren.57_winSkillCheck		
+				trigger_event = tauren.571
+			}
+			88 = { #Fight
+				compare_modifier = {
+					value = scope:duel_value
+					multiplier = -3.5
+					min = -49
+				}
+				custom_tooltip = tauren.57_loseSkillCheck
+				trigger_event = tauren.572
+			}
+		}
+	}
+	
+	#Use the scroll
+	option = {
+		name = tauren.53.d
+		trigger = { learning > 15 }
+		trigger_event = tauren.58
+	}
+}
+#Can't find bandits
+tauren.54 = {
+	type = character_event
+	title = tauren_event_5_name
+	desc = tauren.54.desc
+	theme = skull
+	override_background = wilderness
+	
+ 	left_portrait = {
+ 		character = ROOT
+		animation = anger
+	}
+	right_portrait = {
+		character = scope:companion
+		animation = anger
+	}
+	option = {
+		name = tauren.54.a
+	}
+}
+#Head On battle - Win
+tauren.551 = {
+	type = character_event
+	title = tauren_event_5_name
+	theme = skull
+	desc = tauren.551.desc
+	override_background = bp1_bonfire
+		
+ 	left_portrait = {
+ 		character = ROOT
+		animation = anger
+	}
+	right_portrait = {
+		character = scope:shoeless_bandit
+		animation = fear
+	}
+	
+	option = {
+		name = tauren.561.a
+		add_prestige = 60
+		add_piety = 45
+		if = { limit = { var:wounded_battle = 1 } add_trait = wounded_1 }  
+		scope:kodo_owner = {
+			add_opinion = {
+				modifier = thankful_opinion
+				opinion = 45
+				target = root
+				years = 5
+			}
+		}
+		scope:shoeless_bandit = {
+			death = {
+				death_reason = death_battle
+				killer = root
+			}
+		}
+		
+	}
+	option = {
+		name = tauren.561.b
+		add_gold = 50
+		add_prestige = 25
+		add_piety = -10
+		if = { limit = { var:wounded_battle = 1 } add_trait = wounded_1 }  
+		scope:shoeless_bandit = {
+			death = {
+				death_reason = death_battle
+				killer = root
+			}
```

Comment:
instead of death use `override_death_killer_effect = { death_reason = killer = }` if you have killer
`override_death_effect = { death_reason = }` if you dont have killer. Check and fix in all places

Reply:
Any other notion you'd like to use for imprisonments? Or the imprison = {} works?

Reply:
we use vanilla one, we replaced only `death = ` with our own `scripted_effect`